app/views/declaration/nact_code_add_first.scala.html

```diff
@@ -75,7 +75,7 @@
                 RadioItem(
                     id = Some("code_yes"),
                     value = Some(YesNoAnswers.yes),
-                    content = Text(messages("declaration.transportInformation.containers.yes")),
+                    content = Text(messages("site.yes")),
```

Comment:
Could you remove the old message keys from messages.en?

Reply:
I think that key is still used - by the containers page.  I just copied it in error during the NACT code PR